pkg/namespace/builder.go

```diff
@@ -127,12 +127,20 @@ func setOperation(firstChild *core.SetOperation_Child, rest []*core.SetOperation
 }
 
 // This creates a child for a set operation that references only direct usersets with the parent relation.
+// DEPRECATED
```

Comment:
`Deprecated:`? https://go.dev/blog/godoc

Reply:
Changed to a TODO to remove, otherwise it breaks lint